mocha-multi.js

```diff
@@ -56,12 +60,13 @@ MochaMulti.prototype.done = function (failures, fn) {
     let count = self.reportersWithDone.length;
     debug('Awaiting on %j reporters to invoke done callback.', count);
     const cb = () => {
-      if (--count === 0) {
+      if (count <= 0) {
         debug('All reporters invoked done callback.');
         fn(failures);
       } else {
         debug('Awaiting on %j reporters to invoke done callback.', count);
       }
+      count -= 1;
```

Comment:
this is a logic change, the count used to happen before the condition.

Reply:
ah of course it did.